labs/_templates/labs/base.html

```diff
@@ -7,10 +7,11 @@
     {% endblock %}
   {% endblock %}
 
+{#-
   <div class="content has-text-centered">
     <p class="is-size-7 has-text-grey">
     Version {{ config.VERSION }} built on {{ config.BUILD_TIME }}.
     </p>
   </div>
-
+-#}
```

Comment:
You can remove it completely picking up from the git history if needed (if not forget about, what's tricky), or add an additional explanation why this is commented, so you know when this should become uncommented in the future when the commenting condition changed.